config/src/lib.rs

```diff
@@ -698,6 +706,19 @@ impl Config {
         dirs_next::home_dir().map(|p| p.join(Config::FOUNDRY_DIR_NAME))
     }
 
+    /// Returns the path to `foundry`'s data directory inside the user's data directory
+    /// |Platform | Value                                 | Example                          |
+    /// | ------- | ------------------------------------- | -------------------------------- |
+    /// | Linux   | `$XDG_CONFIG_HOME` or `$HOME`/.config/foundry | /home/alice/.config/foundry
+    /// | | macOS   | `$HOME`/Library/Application Support/foundry   |
+    /// /Users/Alice/Library/Application Support/foundry | | Windows |
+    /// `{FOLDERID_RoamingAppData}/foundry`           | C:\Users\Alice\AppData\Roaming/foundry   |
```

Comment:
nit: nice with a table :smile: can we get it reindented?

Reply:
unfortunately rustfmt breaks this currently, put this in a doc attribute instead